src/datafusion.rs

```diff
@@ -240,7 +240,11 @@ impl ExecutionPlan for DbaseExec {
             if l >= self.limit {
                 break;
             }
-            r = record.ok().unwrap();
+            match record.ok() {
+                Some(record) => r = record,
+                None => continue,
+            };
+
```

Comment:
I think we could use let-else syntax here

```rust
let Some(r) = record else {
    continue;
}
```

Reply:
Done